### DISCUSSION TITLE
Integrations dependencies update

### DISCUSSION THREAD

**Discussion by CreatorHRS** [keycloak/keycloak#20425] [Ideas]
Admin client is using old(legacy) resteasy libs (4.7.7.Final), but new is in project now (6.2.3.Final). So adding this integration to project will cause a bгnch of old dependencies to the project, and some of them have vulnerabilities. I think it will be good to update it. I can try to help with it

**Comment by vmuzikar** [comment]
@CreatorHRS Thanks, feel free to open a PR.

**Reply by slominskir** [reply]
Related: https://github.com/keycloak/keycloak/issues/11026 and https://github.com/keycloak/keycloak/issues/14208
